TINY-9696: Add padding when removing elements if required. (#8681)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

- Table toolbar was visible even if the table was within a noneditable host element. #TINY-9664
- Quickbar toolbars was shown for noneditable contents in a noneditable root. #TINY-9460
- Show the calculated height and width of media embed elements in the `media` plugin dialog. #TINY-8714

> - Removing an image that failed to upload from an empty paragraph would leave the paragraph without the correct padding. #TINY-9696

Reviewer: I initially thought this PR was fixing something to do with style padding when reading the entry but that could just be where my brain went. Might be worth mentioning `br` specifically for clarity.
```suggestion
- Removing an image that failed to upload from an empty paragraph would leave the paragraph without a padding br. #TINY-9696
```

> - Removing an image that failed to upload from an empty paragraph will not leave the paragraph without the correct padding. #TINY-9696

Reviewer 2: ```suggestion
- Removing an image that failed to upload from an empty paragraph would leave the paragraph without the correct padding. #TINY-9696
```
